Source/UI/Message/MessageStack.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MessageStack.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 19/12/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import SwiftUI
+
+struct MessageStack<DataSource>: View where DataSource: MessageList {
+    @ObservedObject
+    var dataSource: DataSource
+
+    @EnvironmentObject
+    private var appController: AppController
```

Comment:
I think we can get rid of the `appController` definition here? I don't think it is needed to pass it down to subviews.

Reply:
👌